### PR TITLE
tweaks

### PR DESCRIPTION
**Pull request by SamVerschueren**
- :arrow_up: got@5.2.0
- :arrow_up: .travis.yml
- Fix tests

### CODE REVIEW

**Comment by arthurvr** [comment]
Thanks! Can you take a look at the failing tests?

**Comment by SamVerschueren** [comment]
Yes, looking into them as we speak :).

File: test.js [hunk 1]
```diff
@@ -2,6 +2,10 @@ import test from 'ava';
 import getStream from 'get-stream';
 import fn from './';
 
+global.Promise = Promise;
```
Comment (SamVerschueren) [inline test.js]: Necessary for Node 0.10. `get-stream` isn't using a Promise ponyfill and thus should be exposed in the global scope to be able to test on 0.10.

**Comment by SamVerschueren** [comment]
@arthurvr fixed

**Comment by sindresorhus** [comment]
Thank you @SamVerschueren :)

**Comment by SamVerschueren** [comment]
No problem :). Thanks for accepting!

**Pull request merged**